### PR TITLE
solution was added

### PR DESCRIPTION
**Pull request by Maksym845**
- [DEMO LINK](https://Maksym845.github.io/react_movies-list-filter/)

### CODE REVIEW

**Review by Yeliseiev — APPROVED** [review]
An interesting solution. But it seems to me that it is better the already filtered list provide to the component

File: src/App.tsx [hunk 1]
```diff
@@ -20,12 +22,19 @@ export const App: React.FC = () => {
...
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList
+          movies={moviesFromServer}
+          searchWord={searchWord}
+        />
```
Comment (sergii-nosachenko) [inline src/App.tsx:37]: No need to use filter logic inside MovieList component. Also your task says you need to create state variable with `visibleMovies` and pass it here as props.

File: src/components/MoviesList/MoviesList.tsx [hunk 2]
```diff
@@ -4,11 +4,17 @@ import { MovieCard } from '../MovieCard';
...
-export const MoviesList: React.FC<Props> = ({ movies }) => (
+export const MoviesList: React.FC<Props> = ({ movies, searchWord }) => (
   <div className="movies">
-    {movies.map(movie => (
+    {movies.filter((movie) => {
+      return (movie.title.toLowerCase()
+        .includes(searchWord.toLowerCase())
+      || movie.description.toLowerCase()
+        .includes(searchWord.toLowerCase()));
+    }).map(movie => (
```
Comment (sergii-nosachenko) [inline src/components/MoviesList/MoviesList.tsx:17]: This function belongs to parent component. Also create create constants f.e. `descriptionNormalized` and use it. Additional lines of code but a huge improvement in readability.

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]

**Review by sergii-nosachenko — APPROVED** [review]
Awesome 🔥🔥🔥

**Pull request closed without merging**